features/visitor_can_see_articles_on_landing_page.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Articles on the landing page
+    As a visitor,
+    In order to choose what article to read,
+    I would like to see the headlines and the first paragraphs of the available articles.
+
+  Scenario: View available articles on the landing page
+    Given the following articles exists
+      | header                | subheader                       | body                    |
+      | Top title             | A breaking news item            | Today at craftacademy   |
```

Comment:
If your User story mentions multiple articles, I would suggest you add more than one to your test background. You want to be sure that you are actually displaying more than one article on the page.